src/applications/claims-status/containers/AskVAPage.jsx

```diff
@@ -15,24 +15,42 @@ class AskVAPage extends React.Component {
     this.setSubmittedDocs = this.setSubmittedDocs.bind(this);
     this.state = { submittedDocs: false };
   }
+
   componentDidMount() {
     document.title = 'Ask for your Claim Decision';
     setUpPage();
   }
+
   componentWillReceiveProps(props) {
     if (props.decisionRequested) {
       props.getClaimDetail(this.props.params.id);
       this.goToStatusPage();
     }
   }
+
   setSubmittedDocs(val) {
     this.setState({ submittedDocs: val });
   }
+
   goToStatusPage() {
     this.props.router.push(`your-claims/${this.props.params.id}`);
   }
+
+  renderBreadcrumbs(claim) {
+    const crumbs = [
+      <a href="/" key="home">Home</a>,
+      <a href="/disability-benefits/" key="disability-benefits">Disability Benefits</a>,
+      <Link to="your-claims" key="claims-home">Track Your Claims and Appeals</Link>,
+      <Link to={`your-claims/${claim.id}`} key="claim-id">Status Details</Link>,
+      <Link to={`your-claims/${claim.id}/ask-va-to-decide`} key="ask-va">Ask for Claim Decision</Link>
+    ];
+
+    return crumbs;
+  }
```

Comment:
I'm not sure if it's worth the trouble or not, but if this basic set of links is shared by most components/containers within this application, it might be worth keeping the application's breadcrumb component, and implementing the Formation Breadcrumbs within that. That way the breadcrumbs of this application remain centralized and if a link changes it only needs to change in a single location.

Reply:
@ncksllvn This is a great call, and I hadn't considered plumbing the Formation breadcrumb through the app's breadcrumb. I will refactor that, and look to retain the logic as it is now.

Reply:
Consolidating shared logic is a good idea. It looks like the links differ slightly, perhaps the shared component could accept props to determine which optional links to include, in order to accommodate this.